### ISSUE TITLE
Can't type hints for collections.Counter

### ISSUE THREAD

**Issue by grasssand** [microsoft/pylance-release#229]
## Environment data

-   Language Server version: 2020.8.0
-   OS and version: Manjaro
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.5

## Expected behaviour

no warning

## Actual behaviour

`Expected class type but received "_Alias"`

## Logs

```

```

## Code Snippet / Additional information

```python
import collections
from typing import ChainMap, Counter, DefaultDict, OrderedDict

a: Counter = collections.Counter()

a1: Counter[str] = collections.Counter()

b: OrderedDict = collections.OrderedDict()

c: ChainMap = collections.ChainMap()

d: DefaultDict = collections.defaultdict()
```
![2020-08-12_09-49](https://user-images.githubusercontent.com/11460505/89966250-513c0000-dc81-11ea-961e-f3655edead4d.png)

**Comment by erictraut** [comment]
Thanks for the bug report. I wasn't aware there was a special alias "Counter" exported by the typing module. I've added support for it. This issue will be fixed in the next release of Pylance.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.8.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202081-13-august-2020
